api.md

```diff
@@ -950,6 +995,47 @@ SELECT device_id, time_bucket('5 minutes', time) as interval,
  alternative to an `ORDER BY time DESC LIMIT 1` clause to find the
  latest value (which will use indexes).
 
+## locf() [](locf)
+
+The `locf` function allows you to carry the last seen value in an aggregation group forward.
+The `locf` function can only be used in an aggregation query with `time_bucket_gapfill`.
+
+#### Required Arguments [](locf-required-arguments)
+
+|Name|Description|
+|---|---|
+| `value` | The value to carry forward (anyelement) |
+| `prev` | The lookup expression for values outside of the gapfill time range |
+
+#### Sample Usage [](locf-examples)
+
+Get the temperature every 5 minutes for each device over the 2 weeks carrying forward the last value for missing readings:
+```sql
+SELECT
+  time_bucket_gapfill('5 minutes', time, now() - interval '2 week', now()) as time,
+  device_id,
+  locf(avg(temperature))
```

Comment:
Same question as for `interpolate` with respect to the description of the required arguments and `prev` being an optional argument?

Reply:
only value is required, the lookup expressions are optional and only needed if you want to look beyond the gapfill time range.